### DISCUSSION TITLE
Two Teslas?

### DISCUSSION THREAD

**Discussion by swa72** [wimaha/TeslaBleHttpProxy#35] [General]
Any hint on how to serve two Teslas (other than to have two Raspberrys with two proxies)?

**Comment by wimaha** [comment]
Just do it. Send the same key to both vehicles. Since every api call includes VIN, it should just work.

**Comment by swa72** [comment]
Ok, so simple do the step "Finally the keys is send to the vehicle. You have to confirm by tapping your NFC card on center console." with the second car?

**Reply by wimaha** [reply]
yes
